pkgs/development/tools/build-managers/apache-maven/builder.sh

```diff
@@ -5,7 +5,7 @@ unpackPhase
 mkdir -p $out/maven
 cp -r $name/* $out/maven
 
-makeWrapper $out/maven/bin/mvn $out/bin/mvn --set JAVA_HOME "$jdk"
+makeWrapper $out/maven/bin/mvn $out/bin/mvn --set JAVA_HOME "$jdk/lib/openjdk"
```

Comment:
@NeQuissimus i'm no java expert but is this folder always going to be openjdk? or would other implementation use different folder (oraclejdk, ...)?